lib/pm.js

```diff
@@ -71,7 +71,15 @@ Pm.prototype.spawn = function(callback) {
 
   mkdirp(this.base);
 
-  var logFd = fs.openSync(this.logFile, 'a');
+  try {
+    fs.truncateSync(this.logFile, 0);
```

Comment:
It was intended to use the 'a' flag, you can see in the snippet above, the 'w' was a relic of troubleshooting :blush: thanks for noticing.